### DISCUSSION TITLE
2 Locations

### DISCUSSION THREAD

**Discussion by HandHeldRehab** [dgreif/ring#1146] [General]
Newby here.

I have 2 different Ring Systems set up and 2 different locations under the same email account.

Ring has these locations separated.

Home Kit has an option to select a specific location.

How can I separate the locations with this plug in?

I want to get notifications from either location, but I want the ability to sort them by locations also.

**Comment by dgreif** [comment]
You should be able to create two copies of the config in your homebridge config. You can then use the `locationIds` option to filter to a single Ring location per config. You will also want to use child bridges so that you can add them to HomeKit independently. This _should_ allow you to add each child bridge to the desired Home in HomeKit.
